tests/integration/targets/rds_cluster_modify/tasks/main.yaml

```diff
@@ -4,7 +4,16 @@
       aws_access_key: '{{ aws_access_key }}'
       aws_secret_key: '{{ aws_secret_key }}'
       security_token: '{{ security_token | default(omit) }}'
+    group/amazon.cloud.aws:
```

Comment:
Why is this needed here?

Reply:
for the tasks in integration tests that use `amazon.cloud` collection
like https://github.com/ansible-collections/amazon.aws/pull/1705/files#diff-91943e165e49a6da17ee2d4839a338f90e72c809e45c6772437bf3f62d383664R5-R12

Reply:
I think you need to use the collections keyword to list the amazon.cloud collection.

Reply:
I don't think that is needed, from my testing it looks like, if I remove that part, the the tasks that use `amazon.cloud` fail with `credentials not found`

```
TASK [rds_cluster_modify : Create rds global database] *************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: botocore.exceptions.NoCredentialsError: Unable to locate credentials
fatal: [testhost]: FAILED! => {"boto3_version": "1.22.0", "botocore_version": "1.27.82", "changed": false, "msg": "Failed to modify resource: Unable to locate credentials", "resource_actions": ["cloudcontrolapi:GetResource"]}
```